build.gradle

```diff
@@ -8,62 +8,23 @@ buildscript {
 }
 
 plugins {
-  id 'java-library'
-  id 'signing'
-  id 'maven-publish'
-  id 'idea'
   id 'eclipse'
   id 'project-report'
   id 'com.diffplug.spotless' version '6.22.0'
   id "com.google.protobuf" version "0.9.4"
-  id 'io.github.gradle-nexus.publish-plugin' version "1.3.0"
+  id 'org.wiremock.tools.gradle.wiremock-extension-convention' version '0.1.2'
 }
 
-repositories {
-  mavenLocal()
-  mavenCentral()
-}
-
-group 'org.wiremock'
+group 'org.wiremock.extensions'
```

Comment:
If we're going to move it we should probably drop a tombstone on the old coordinates